.github/workflows/patch.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
Should we use `ubuntu-latest` instead?

Reply:
Only reason I didn't pick latest is as I thought it will auto-bump to 22.04 _immediately_.  But it seems Github takes some time after the official release, so it should be okay.